src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const envConnectHost = "OP_CONNECT_HOST";
+export const envConnectToken = "OP_CONNECT_TOKEN";
+export const envServiceAccountToken = "OP_SERVICE_ACCOUNT_TOKEN";
+export const envManagedVariables = "OP_MANAGED_VARIABLES";
+
+export const authErr = `(${envConnectHost} and ${envConnectToken}) or ${envServiceAccountToken} must be set`;
```

Comment:
This may be a nitpick, but can we make this error clearer by making it a bit more specific? For example:

`Authorization error with environment variables: you must set either 1) ${envServiceAccountToken}, or 2) both ${envConnectHost} and ${envConnectToken}.`